.github/workflows/manual-deploy-dev-staging.yml

```
@@ -18,11 +18,96 @@ on:
 env:
   DEVOPS_CHANNEL_ID: C37M86Y8G #devops-deploys
   VETS_WEBSITE_CHANNEL_ID: C02V265VCGH # status-vets-website
+  BUILD_ENV_DEV: vagovdev
+  BUILD_ENV_STAGING: vagovstaging
 
 jobs:
+  build:
+    name: Build
+    runs-on: self-hosted
+
+    env:
+      NODE_EXTRA_CA_CERTS: /etc/ssl/certs/VA-Internal-S2-RCA1-v1.cer.pem
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

I think `ref` needs to be set here, otherwise this will default to checking out the sha of the latest commit in the branch, which won't match the sha specified in the input. Have you tried testing with an older sha from this branch?
```suggestion
        uses: actions/checkout@v2
        with:
          ref: ${{ github.event.inputs.commit_sha }}
```

---

good thought process, makes sense.